src/ui/screens/settings/SyncSettingsScreen.tsx

```diff
@@ -84,6 +91,21 @@ export const SyncSettingsScreen: FC = () => {
           subtext="Syncs all records currently stored in sync queue"
           onPress={() => dispatch(SyncAction.syncAll(loginUrl, sensorUrl, temperatureLogUrl, temperatureBreachUrl, username, password))} 
         />
+        {
+          isPassiveSyncEnabled ? (
+            <SettingsButtonRow
+              label="Stop sync scheduler"
+              subtext={`Stops sync scheduler (last sync: ${moment.unix(lastSync).format(FORMAT.DATE.STANDARD_DATE)})`}
```

Comment:
I think we should (in the future?) not import `moment` and instead use `formatService` to do all the formatting of dates etc.

moment is essentially getting deprecated and is a good reminder that any dependency might need to be swapped out!

Reply:
Issue created (#85).